nengo/tests/test_node.py

```diff
@@ -139,25 +139,31 @@ def test_circular(Simulator, seed):
     assert np.allclose(sim.data[a_p], sim.data[b_p])
 
 
-def test_function_args_error(Simulator):
+def test_outputparam_errors(Simulator):
     with nengo.Network() as model:
+        # type errors
```

Comment:
Maybe move the comment down to where actually a type error occurs and change this line to something like `# valid values`?